platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/inc/psa/crypto_extra.h

```diff
@@ -54,15 +56,17 @@ extern "C" {
  *                              for, in addition to the algorithm set with
  *                              psa_set_key_algorithm().
  *
- * \warning Setting an enrollment algorithm is not recommended, because
- *          using the same key with different algorithms can allow some
- *          attacks based on arithmetic relations between different
- *          computations made with the same key, or can escalate harmless
- *          side channels into exploitable ones. Use this function only
- *          if it is necessary to support a protocol for which it has been
- *          verified that the usage of the key with multiple algorithms
- *          is safe.
+ * \deprecated  This is for backward compatibility only.
+ *              Setting an enrollment algorithm is not recommended, because
+ *              using the same key with different algorithms can allow some
+ *              attacks based on arithmetic relations between different
+ *              computations made with the same key, or can escalate harmless
+ *              side channels into exploitable ones. Use this function only
+ *              if it is necessary to support a protocol for which it has been
+ *              verified that the usage of the key with multiple algorithms
+ *              is safe.
```

Comment:
The explanation around the deprecation already exists. "backward compatibility" refers to previous releases of Mbed OS.